Apps/AdminWebClient/src/ClientApp/src/views/Communication.vue

```diff
@@ -86,13 +108,126 @@ export default class CommunicationView extends Vue {
     message: ""
   };
 
+  private tableHeaders: any[] = [
+    {
+      text: "Subject",
+      value: "subject",
+      width: "30%"
+    },
+    {
+      text: "Effective On",
+      value: "effectiveOn",
+      width: "20%"
+    },
+    {
+      text: "Expires On",
+      value: "expiresOn",
+      width: "20%"
+    },
+    {
+      text: "Text",
+      value: "text",
+      width: "40%"
+    }
+  ];
+
+  private bannerList: Communication[] = [];
+
   private communicationService!: ICommunicationService;
 
   mounted() {
     this.communicationService = container.get(
       SERVICE_IDENTIFIER.CommunicationService
     );
     this.clearForm();
+    this.loadBannerList();
+  }
+
+  private loadBannerList() {
```

Comment:
Please make sure you create a work item to cover this work.

Reply:
Created 8578 "replace the mock service with the real service" to cover this work per @sslaws 's comment